astropy/coordinates/builtin_frames/equatorial.py

```diff
@@ -61,16 +61,16 @@ class TETE(BaseRADecFrame):
     TETE coordinates are often referred to as "apparent" coordinates, or
     "apparent place". TETE is the apparent coordinate system used by JPL Horizons
     and is the correct coordinate system to use when combining the right ascension
-    with local apparent sidereal time to calculate the hour angle.
+    with local apparent sidereal time to calculate the apparent hour angle.
```

Comment:
I'd like to see:
```
with local apparent sidereal time to calculate the apparent (TIRS) hour angle
```
here. While in the context of TETE, this is the apparent hour angle, it is the same as the hour angle that would be computed in the CIRS frame, so it's generally the TIRS hour angle.

Reply:
I am on it

Reply:
I have added (TIRS) at line 64, please review it.